CHANGELOG.md

```diff
@@ -191,20 +190,20 @@ The binaries of v2.1.0 have been published, but not the other artifacts (AUR, Do
 ### v2.0.0
 
 1. Enhancements
-   * 🌟 New `golangci-lint fmt` command with dedicated [formatter configuration](https://golangci-lint.run/welcome/quick-start/#formatting)
-   * ♻️ New `golangci-lint migrate` command to help migration from v1 to v2 (cf. [Migration guide](https://golangci-lint.run/product/migration-guide/#command-migrate))
-   * ⚠️ New default values (cf. [Migration guide](https://golangci-lint.run/product/migration-guide/))
-   * ⚠️ No exclusions by default (cf. [Migration guide](https://golangci-lint.run/product/migration-guide/#issuesexclude-use-default))
-   * ⚠️ New default sort order (cf. [Migration guide](https://golangci-lint.run/product/migration-guide/#outputsort-order))
-   * 🌟 New option `run.relative-path-mode` (cf. [Migration guide](https://golangci-lint.run/product/migration-guide/#runrelative-path-mode))
-   * 🌟 New linters configuration (cf. [Migration guide](https://golangci-lint.run/product/migration-guide/#linters))
-   * 🌟 New output format configuration (cf. [Migration guide](https://golangci-lint.run/product/migration-guide/#output))
-   * 🌟 New `--fast-only` flag (cf. [Migration guide](https://golangci-lint.run/product/migration-guide/#lintersfast))
+   * 🌟 New `golangci-lint fmt` command with dedicated [formatter configuration](https://golangci-lint.run/docs/welcome/quick-start/#formatting)
```

Comment:
Is it possible to avoid breaking existing links?

There are many references to the Golangci-lint documentation, such as:
- https://sourcegraph.com/search?q=context:global+golangci-lint.run/usage&patternType=standard&sm=1
- https://github.com/search?q=golangci-lint.run%2Fusage&type=code

Reply:
Not really: I don't want to set up a server or proxy.

The links will be broken; it's expected.

Reply:
Perhaps we should write an article that lists old and new links? Like:

| Old | New |
|-----|-----|
| https://golangci-lint.run/usage/linters/ | http://golangci-lint.run/docs/linters/ |
| https://golangci-lint.run/usage/formatters/ | http://golangci-lint.run/docs/formatters/ |
| https://golangci-lint.run/usage/false-positives/ | http://golangci-lint.run/docs/linters/false-positives/ |
| ... | ... |

Or provide a simple script (uses `sed`) that automatically updates links across the codebase.